cli/src/config.rs

```diff
@@ -324,7 +324,17 @@ fn env_base() -> config::Config {
         builder = builder.set_override("ui.color", "never").unwrap();
     }
     if let Ok(value) = env::var("PAGER") {
-        builder = builder.set_override("ui.pager", value).unwrap();
+        if value == "less" {
+            // Many systems ship with "R" as the only setting in $LESS, which provides
+            // a less than ideal experience with jj, since they won't see the log history
+            // persist on the screen. While slightly inconsistent to ignore that setting,
+            // it should provide a better "out-of-box experience". If a user truly wants
+            // to change the values that are passed to "less", they can set that in the
+            // ui.pager jj config.
+            builder = builder.set_override("ui.pager", "less -FRX").unwrap();
```

Comment:
Maybe we should simply not do anything here and inherit the default value instead:
https://github.com/martinvonz/jj/blob/bcdb9beb6ce5ba625ae73d4839e4574db3d9e559/cli/src/config/misc.toml#L9

Reply:
Seems fine to me, basically if your pager is set to less, you get the default settings instead :+1:

Reply:
Yes, and we don't need to duplicate the `LESSCHARSET` bit. OTOH, if we later change the default to something else, you'll get that new default if you say `PAGER=less`. Hopefully we'll notice that :)